README.md

```diff
@@ -0,0 +1,173 @@
+# RatSLAM ROS (OpenRatSLAM)
+
+[![Build Status](https://travis-ci.org/sem23/ratslam.svg?branch=ratslam_ros)](https://travis-ci.org/sem23/ratslam)
+
+## Package Summary
+
+This package contains the RatSLAM sources including ROS wrappers. The package provides a biological inspired monocular
+vision SLAM (Simultaneous Localization and Mapping) system. Using this system you can create a topological map of the
+environment, which means there are just paths and **no** gridmap, like an occupancy grid. As a gimmick, the system is
+able to perfom pathplanning on the current map.
+
+Authors Documentation:<br>
+The ROS version of RatSLAM has been designed as a more modular version than the other library in this project - the algorithm has been divided into 3 individual RatSLAM ROS nodes and one visual odometry node. This version is referred to as OpenRatSLAM.
+
+OpenRatSLAM is based on the RatSLAM C++ library written by David Ball and Scott Heath, with support and additions by Michael Milford. The RatSLAM algorithm was originally designed and implemented by Michael Milford and Gordon Wyeth.
+
+OpenRatSLAM has only been tested on Ubuntu and as ROS support for platforms other than Linux is still limited, this will probably not change.
+
+* Maintainer status: maintained
+* Maintainer: Peter Rudolph <semael23 AT gmail DOT com>
+* Authors: Michael Milford and Gordon Wyeth (RatSLAM), David Ball and Scott Heath (RatSLAM C++), David Ball (OpenRatSLAM)
+* License: GNU General Public License v3.0
+* Source: git https://github.com/sem23/ratslam.git (branch: ratslam_ros)
+
+## 1. External Documentation
+
+Further documentation can be found on ROS wrapper authors [git](https://github.com/davidmball/ratslam).
```

Comment:
Add in the publication: David Ball, Scott Heath, Janet Wiles, Gordon Wyeth, Peter Corke, Michael Milford: OpenRatSLAM: an open source brain-based SLAM system, Autonomous Robots, 2013 and link https://link.springer.com/article/10.1007%2Fs10514-012-9317-9